docs/README.md

```diff
@@ -100,6 +100,28 @@ Now, you are able to use all blocks exported by the `store-newsletter` app. Chec
 | `url`     | `string` | The link's URL.             | `undefined`   |
 | `text`    | `string` | Text displayed on the link. | `undefined`   |
 
+### `newsletter-hidden-field` props
+
+| Prop name | Type     | Description                 | Default value |
+| --------- | -------- | --------------------------- | ------------- |
+| `customFields`     | `CustomField[]` | An array of `CustomField` objects.             | `undefined`   |
+
+The `CustomField` object enables you to specify a certain custom user field that should always be sent along with a user's newsletter subscription. It has the following format:
+
+```ts
+interface CustomField {
+  fieldName: string,
+  dynamicValue?: "bindingId" | "bindingUrl",
+  value?: string | number | boolean
+}
+```
+
+- `fieldName` should match the field's name in your Master Data user schema.
+
+- `dynamicValue` currently only supports `bindingId` and `bindingUrl`, and both these values will be calculated by `newsletter-hidden-field` before sending the newsletter subscription data.
+
+- `value` is just a static value that will always be the same, and will always be sent when users submit the newsletter form. You should either use `value` or `dynamicValue`, never both.
+
```

Comment:
```suggestion
### `newsletter-hidden-field` props

| Prop name | Type | Description | Default value |
| :---------: | :----: | :----------: | :-------------: |
| `customFields` | `object` | Enables you to specify a certain custom user field that should always be sent along with a user's newsletter subscription. | `undefined` |

- **`customFields` object:**

| Prop name | Type | Description | Default value |
| :---------: | :----: | :----------: | :-------------: |
| `fieldName` | `string` | Text string that should match the field's name in your Master Data user schema. | `undefined` | 
|  `dynamicValue` | `enum` | Possible values are: `bindingId` and `bindingUrl`. Both of these values will be calculated by the `newsletter-hidden-field` block before sending the newsletter subscription data. | `undefined` |
| ` value` | `string` | Static value sent to users when they submit the newsletter form. *Caution*: the value will always be the same. | `undefined` | 

:warning: *The `value` and `dynamicValue` props are conflicting and must not be declared together.*

```

Reply:
I don't think I fully understood the idea. The descriptions are still confused to me, I think we can clarify the modus operandi behind this object!

Reply:
The new block is missing in the block list (at the top of the documentation)